src/global_distrib/mod_global_distrib_mapping_redis.erl

```diff
@@ -71,7 +71,7 @@ put_session(Jid) ->
     ets:insert(?JIDS_ETS, {Jid}),
     do_put(Jid, opt(local_host)).
 
--spec get_session(Jid :: binary()) -> {ok, Host :: jid:lserver()} | error.
+-spec get_session(Jid :: binary()) -> {ok, Host :: binary()} | error.
```

Comment:
Why the type change from `jid:lserver` to `binary`? I know that in the end it's the same. Using the type `jid:lserver` can at least tell the developer that the value is expected to be stringprepped.

Reply:
I believe it was because unfortunately dialyzer was complaining about that, for some really strange reason it was not liking it 😕

Reply:
The specs in `mod_global_distrib_mapping` and `mod_global_distrib_mapping_redis` were all a bit messy, there's three different things in three different places, so to make dialyzer happy I took the path of least diff 🤷‍♂